core/src/services/ftp/backend.rs

```diff
@@ -221,13 +223,21 @@ impl bb8::ManageConnection for Manager {
 
     async fn connect(&self) -> Result<Self::Connection, Self::Error> {
         let stream = ImplAsyncFtpStream::connect(&self.endpoint).await?;
-        // switch to secure mode if ssl/tls is on.
         let mut ftp_stream = if self.enable_secure {
+            // Create a root certificate store
+            let root_store = RootCertStore::empty();
+            // Optionally, add certificates (e.g., system certs or custom ones)
+            // For now, we'll leave it empty as an example
```

Comment:
I'm guessing we need to add system credentials at the very least?

Reply:
seems  no need follow the old `default` will change the comment